snowplow/iglu-client-embedded/schemas/com.metabase/downloads/jsonschema/1-0-0

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Downloads results from questions and dashcards",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "downloads",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "type": "string",
+      "enum": [
+        "download_results_clicked"
+      ],
+      "description": "The type of event being recorded."
+    },
+    "resource_type": {
+      "type": [
+        "string",
```

Comment:
I was thinking about that too at first, but it seems like we try to change the schemas as little as possible and we already have typescript types for those fields.
I'm not sure if having enums makes it more performant, it could just be that we could miss some data if it's malformed

Reply:
Changing the schema is fine in my opinion. It's better than keeping the schema as is, but keep changing what we send in isn't it? That would make tracking changes a lot harder, but on the otherhand, changing schema is very obvious that we change the value we're sending in. Making an enum in the schema forces us to be more deliberate about what we're tracking.

Reply:
That's a good point, I asked on the data channel here: https://metaboat.slack.com/archives/C01MC2CKW2G/p1722349249986189

Reply:
https://github.com/metabase/metabase/pull/46132/commits/bbd224fdd932d25cf3cc4d5b6107f0532fccc380
@WiNloSt I updated the schema to use enums, let me know if it looks good

Reply:
should you make `resource_type` an enum too?

Reply:
Totally, I have no idea how I missed it 😅
f7ae4fade53b9be47d12f0523e39bbc29cb014e0